developer-docs-site/docs/sdks/aptos-sdk-overview.md

```diff
@@ -0,0 +1,144 @@
+
+1. The transportation layer.
+2. The core SDK layer.
+3. An optional application layer. 
+
+The transportation layer is responsible for sending payloads to the REST API endpoints.
+
+The core SDK layer exposes the functionalities needed by most applications, including:
+
+- The key generation.
+- Transaction signing and submission.
+- Transaction status querying, and
+- Various kinds of information retrieval.
+
+The optional application layer provides built-in support for **NFT token** API. 
+
+:::tip
+You can also use this NFT token API as an example before you start developing your own application APIs using the SDK.
+:::
+
+### OpenAPI client
+
+The OpenAPI client is a set of classes that are generated based on the Aptos REST API spec. See the [Typescript SDK OpenAPI definition](https://aptos-labs.github.io/ts-sdk-doc/).
+
+### Aptos Account
+
+The [class AptosAccount](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html) provides the methods for:
+
+- Generating Ed25519 key pairs.
+- Signing a bytes buffer with an Ed25519 public key, and 
+- Deriving initial account addresses from the public keys.
+
+### BCS Library
+
+A subset of BCS standards implemented in Typescript.
+
+### Transaction builder
+
+The transaction builder contains the Typescript types for constructing the transaction payloads. The transaction builder within the Typescript SDK supports the following transaction payloads:
+
+1. ScriptFunction
+2. Script
+3. ModuleBundle
+
+### Aptos Client
+
+The [class AptosClient](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosClient.html) exposes the methods for retrieving the account resources, transactions, modules and events.
+
+In addition, the `AptosClient` component supports two methods for transaction signing and submission.
+
+1. Submitting transactions in JSON format, which delegates the signing message (input to the signing function) creation to the API server. This is applicable when using the REST API and the Aptos server to generate the signing message, the transaction signature and submit the signed transaction to the Aptos Blockchain. See the tutorial [Your First Transaction](/tutorials/first-transaction.md).
+2. Submitting transactions in BCS format, which prepares and signs the raw transactions on the client-side. This method leverages the BCS Library and Transaction Builder for constructing the transaction payloads. See the guide [Creating a Signed Transaction](/guides/sign-a-transaction.md).
+
+:::tip
+
+The second method, i.e., in BCS format, is the recommended way for submitting transactions to the Aptos Blockchain.
+
+:::
+
+### Token Client
+
+The class [TokenClient](https://aptos-labs.github.io/ts-sdk-doc/classes/TokenClient.html) provides methods for creating and querying the NFT collections and tokens.
+
+## Validation for Transaction Builder and BCS
+
+The Transaction Builder and the BCS are used to assemble and serialize the transaction payloads for signing and submission.
```

Comment:
just BCS -- binary canonical serialization
it might be useful to link to the upstream bcs
https://github.com/diem/bcs